hacking/test-module

```diff
@@ -178,6 +181,7 @@ def ansiballz_setup(modfile, modname, interpreters):
 
     cmd = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
     out, err = cmd.communicate()
+    out, err = to_text(out), to_text(err)
```

Comment:
There's one cornercase here.  Later on we're using a portion of ```out``` (```lines[1]```) to construct debug_dir which is part of a file path.  The default of to-text() for undecodable bytes is to replace the undecodable bytes with a replacement character.  We don't want that to happen with debug_dir.  So we should probably change the error handler for out like this:

``` python
out, err = to_text(out, errors='surrogate_or_strict'), to_text(err)
```
If out can't be decoded, it will then throw an exception.  err it doesn't matter since ultimately, the only thing we do with that is print it to the screen.

We could instead change the file paths to be byte strings all the way through the code but that's a large change for something that won't happen very frequently in code that's only run for testing modules.  So I think using to_text is fine but we just want to throw an error here if someone is using non-utf8 file paths.